source/documentation/virtual_machine_management_guide/topics/Optimizing_Live_Migration.adoc

```diff
@@ -3,6 +3,11 @@
 
 Live virtual machine migration can be a resource-intensive operation. To optimize live migration, you can set the following two options globally for every virtual machine in an environment, for every virtual machine in a cluster, or for an individual virtual machine.
 
+[NOTE]
+====
+For cluster levels 4.3 or later, auto converge is enabled by default for all built-in migration policies, and migration compression is enabled by default for only the `Suspend workload if needed` migration policy. You can change these parameters by modifying the `MigrationPolicies` configuration value, and when adding a new migration policy.
+====
+
```

Comment:
I'm not sure you need to state "and when adding a new migration policy" because AFAIK, you can always change defaults when you configure an entity. Up to you, of course.